test-requirements.txt

```diff
@@ -5,9 +5,15 @@ coverage
 docker-compose
 flake8
 mock
+mxnet-model-server
 pytest
 pytest-cov
 pytest-xdist
+python-dateutil==2.8.0
```

Comment:
When I tried building my inference changes on top of this branch, I noticed that during the installation of the final Dockerfile.cpu, there's an error regarding the python-dateutil version conflicting with botocore when installing requirements.txt. Since this also affects the container build, can you move this to requirements.txt instead of having it in here?